Feat: Add logout function

hannadoolsetnet opened the pull request:
## 로그아웃 기능 추가

사용자 메뉴의 로그아웃 버튼 클릭 시 local storage의 사용자 아이디 정보와 쿠키를 삭제하여 로그아웃 처리하는 기능입니다.

hannadoolsetnet committed:
Feat: Add logout function

imjaehoo commented on src/components/AppHeader/UserTools/UserMenuButton.js:
```diff
@@ -14,14 +14,10 @@ export default function UserMenuButton({ onClick }) {
     <>
       <Button variant={ICON} onClick={handleDropDown} children={UserMenuIcon} />
-      {toggleDropDown && <UserMenuDropdown onClick={handleLoginStatus} />}
+      {toggleDropDown && <UserMenuDropdown handleLoginStatus={onClick} />}
```
오잉 뭔가 뒤바뀐 것 같아요

imjaehoo commented on src/components/AppHeader/UserTools/UserMenuDropdown.js:
```diff
@@ -7,33 +7,23 @@ import firebaseAPI from "../../../api/firebase";
-    onClick(false);
-  };
-
+export default function UserMenuDropdown({ handleLoginStatus }) {
```
여기선 onClick으로 받고
그 onClick으로 넘어오는 함수를 쓰는 입장에서 더 명확하게 쓸 수 있도록 해주고 싶다면 이쪽에서
```
function handleLoginStatus() {
  ...
  onClick()
}
```
이렇게 함수를 만들어주면 될 거예요. 이걸 하고 싶으셨던 게 맞다면!

hannadoolsetnet replied:
반영하여 수정 완료했습니다! 감사합니다!🙏🏻

hannadoolsetnet committed:
Refactor: Modify code style

  - add keys to userMenu buttons
  - remove unused variable
  - modify props names and condition executing handler function
